src/js/views/Certificates/layout/AssociateToDeviceModal.jsx

```diff
@@ -112,12 +114,14 @@ const AssociateDevicesModal = ({ isOpen, certificate, handleHideDevicesToAssocia
 
   return (
     <Dialog open={isOpen} onClose={handleHideDevicesToAssociateModal} maxWidth='lg' fullWidth>
+      <Helmet title={`${t('certificates:associateToDevice')} • ${t('common:dojotPageTitle')}`} />
```

Comment:
I don't know if we should change the document title when opening a modal, because the document title is registered in the browser history, but we can't (for now) navigate to the page with the modal opened. The same thing is valid for the device creation steps, we can't open the page in a specific step, so the document title should be only the page name for all steps.

@luisfelipefurlan  What do you think?`

Reply:
I agree with you.
As the modal is not reachable via URL, it should not change the page title. I believe that no modal should change the title.